management/management_api/tenants/tenants_utils.py

```diff
@@ -301,10 +301,4 @@ def list_tenants(id_token):
     for item in namespaces.to_dict()['items']:
         if item['status']['phase'] != TERMINATION_IN_PROGRESS:
             tenants.append(item['metadata']['name'])
-    if not tenants:
-        message = "There are no tenants present on platform\n"
-        logger.info(message)
-        return message
-    message = f"Tenants present on platform: {tenants}\n"
-    logger.info(message)
-    return message
+    return tenants
```

Comment:
Maybe instead of only list of tenants in `data`, we could also have a user friendly `message` in the JSON? Like it was earlier?

Reply:
I think the json should only contain data and the client (e.g cli or web ui) should add the logic how to display/print the data. This would make e.g. the client side language localization much simpler.